Tests/XCUITests/DownloadsTests.swift

```diff
@@ -109,41 +99,33 @@ class DownloadsTests: BaseTestCase {
         navigator.goto(BrowserTabMenu)
         navigator.goto(LibraryPanel_Downloads)
         mozWaitForElementToExist(app.tables["DownloadsTable"])
-        if processIsTranslatedStr() == m1Rosetta {
-            throw XCTSkip("swipeLeft() does not work on M1")
-        } else {
-            deleteItem(itemName: testFileNameDownloadPanel)
-            mozWaitForElementToNotExist(app.tables.cells.staticTexts[testFileNameDownloadPanel])
-            // After removing the number of items should be 0
-            checkTheNumberOfDownloadedItems(items: 0)
-        }
+        deleteItem(itemName: testFileNameDownloadPanel)
+        mozWaitForElementToNotExist(app.tables.cells.staticTexts[testFileNameDownloadPanel])
+        // After removing the number of items should be 0
+        checkTheNumberOfDownloadedItems(items: 0)
     }
 
     // https://testrail.stage.mozaws.net/index.php?/cases/view/2306901
     func testShareDownloadedFile() throws {
         downloadFile(fileName: testFileName, numberOfDownloads: 1)
         navigator.goto(BrowserTabMenu)
         navigator.goto(LibraryPanel_Downloads)
-        if processIsTranslatedStr() == m1Rosetta {
-            throw XCTSkip("swipeLeft() does not work on M1")
+        let shareButton = app.tables.buttons.staticTexts["Share"]
+        app.tables.cells.staticTexts[testFileNameDownloadPanel].swipeLeft()
+        mozWaitForElementToExist(shareButton)
+        XCTAssertTrue(shareButton.exists)
```

Comment:
@jjSDET Do we need to call `snapshot()` before we can use `XCTAssert` commands?

Reply:
I would say to start using it in those tests that are flaky, to be sure the assertion is done once the element is for sure there